packages/x-data-grid/src/tests/dataSource.DataGrid.test.tsx

```diff
@@ -160,8 +160,8 @@ describeSkipIf(isJSDOM)('<DataGrid /> - Data source', () => {
 
     setProps({
       paginationModel: { page: 1, pageSize: 10 },
-      sortModel: [{ field: 'name', sort: 'asc' }],
```

Comment:
Adding the warning surface this hidden issue in the test, it was trying to set the `sortModel` field to `name` which was not there.